packages/botonic-plugin-knowledge-bases/src/index.ts

```diff
@@ -31,21 +31,13 @@ export default class BotonicPluginKnowledgeBases implements Plugin {
       sources
     )
 
-    const responseSources = response.data.sources.map(source => {
-      return {
-        knowledgeBaseId: source.knowledge_base_id,
-        knowledgeSourceId: source.knowledge_source_id,
-        knowledgeChunkId: source.knowledge_chunk_id,
-      }
-    })
-
```

Comment:
should the changes in this PR go in a 0.29.0 release? this is a breaking change. Maybe we can update just this plugin to 0.29.0 to maintain compatibility if needed